libfreerdp/core/rdp.c

```diff
@@ -1384,7 +1393,7 @@ static int rdp_recv_tpkt_pdu(rdpRdp* rdp, wStream* s)
 				          "pduType %s not properly parsed, %" PRIdz
```

Comment:
Perhaps you could include my patch (gist sent on IRC), as it fixes the fact that you can announce a small `pduLength` but decoding functions will be able to read more than `pduLength` (because they do checks with `Stream_GetRemainingLength(mainStream)`)